go/pkg/rexec/rexec.go

```diff
@@ -88,9 +94,12 @@ func (ec *Context) downloadStream(raw []byte, dgPb *repb.Digest, write func([]by
 		if err != nil {
```

Comment:
If we're going to support reading from an offset here, perhaps we should use the corresponding [ByteStream feature](https://github.com/googleapis/googleapis/blob/511319c6af9172bf12de3e80672b8109c49efd29/google/bytestream/bytestream.proto#L107), rather than reading the entire blob and then filtering it.

Reply:
Good point.